src/components/ecology.jsx

```diff
@@ -3,42 +3,44 @@ import API from "./api";
 import Overview from "./overview";
 
 export default class Ecology extends React.Component {
-  renderAPI(source) {
```

Comment:
Right, I could refactor the other way where I work with the variable being passed. Just thought it was weird we were passing the prop as a variable, but then using the prop as the source. And yeah, needs some kind of return in the method in its current form.

Let me know if the way I did it is weird, and I can refactor. It might be goofy doing the check to see if it exists outside of the method.